### PR TITLE
Feature/topiks

### PR DESCRIPTION
**Pull request by zelenyid**
Added topics, posts and authorization to forum site

### CODE REVIEW

File: app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class RegistrationsController < Devise::RegistrationsController
```
Comment (tiya-lorie) [inline app/controllers/registrations_controller.rb:1]: Check is it used. If not please delete

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class UsersController < ApplicationController
+  def sign_up
+    @user = User.create(user_params)
+    if @user.valid?
+      token = encode_token({ user_id: @user.id })
+      render json: { user: @user, token: token }
```
Comment (tiya-lorie) [inline app/controllers/users_controller.rb]: Do you use json?

Comment (zelenyid) [reply app/controllers/users_controller.rb]: No, I've deleted this file

File: app/controllers/registrations_controller.rb (hunk 1 above)
Comment (zelenyid) [reply app/controllers/registrations_controller.rb:1]: Without this class will be problems with registration and updating user profile

**Review by tiya-lorie — APPROVED** [review]

**Pull request merged**